src/peft/import_utils.py

```diff
@@ -63,3 +63,19 @@ def is_torch_tpu_available(check_device=True):
                 return False
         return True
     return False
+
+
+def is_auto_awq_available():
+    if importlib.util.find_spec("awq") is not None:
+        # TODO: to change it to 0.2.0 after the AutoAWQ release
```

Comment:
@BenjaminBossan I 'll change that to `0.2.0` before merging right after auto-awq's release